### PR TITLE
Exclude grade field; add help_text for xp adjustment

### LINKED ISSUE #1230: Remove Grade field from student-course form; add helptext for XP Adjustment
Should not be there.  Also, add help test explaining XP adjustment fields.

image

### PR DESCRIPTION
**Pull request by yujinyuz**
Please ensure you are familiar with our Pull Request Description expectations described here: https://www.pullrequest.com/blog/writing-a-great-pull-request-description/
### What?

Remove grade field

Closes #1230 

### Why?

This field should not be here in the first place

### How?

By adding grade_fk to the excluded fields

### Testing?

Manually tested

### Screenshots (if front end is affected)

image

### Review request
@tylerecouture

## Summary by CodeRabbit

- **Refactor**
	- Improved clarity in the course enrollment form by adjusting default parameter values.
- **Documentation**
	- Updated help text for XP adjustment fields in the course management system for better user guidance.

### CODE REVIEW

File: src/courses/models.py [hunk 1]
```diff
@@ -512,7 +512,7 @@ class CourseStudent(models.Model):
     course = models.ForeignKey(Course, on_delete=models.PROTECT, null=True)
     # grade is deprecated, shouldn't be used anywhere any more
     grade_fk = models.ForeignKey(Grade, verbose_name="Grade", on_delete=models.SET_NULL, null=True, blank=True)
-    xp_adjustment = models.IntegerField(default=0)
+    xp_adjustment = models.IntegerField(default=0, help_text="Adjustment to XP for this course")
     xp_adjust_explanation = models.CharField(max_length=255, blank=True, null=True)
```
Comment (tylerecouture) [inline src/courses/models.py:516]: help_text="An optional reminder why an XP Adjustment was provided."

**Review by tylerecouture — CHANGES_REQUESTED** [review]

File: src/courses/models.py [hunk 2]
```diff
@@ -512,7 +512,7 @@ class CourseStudent(models.Model):
     course = models.ForeignKey(Course, on_delete=models.PROTECT, null=True)
     # grade is deprecated, shouldn't be used anywhere any more
     grade_fk = models.ForeignKey(Grade, verbose_name="Grade", on_delete=models.SET_NULL, null=True, blank=True)
-    xp_adjustment = models.IntegerField(default=0)
+    xp_adjustment = models.IntegerField(default=0, help_text="Adjustment to XP for this course")
```
[the remark this answers is not in the document]
Comment (yujinyuz) [reply src/courses/models.py]: done

File: src/courses/models.py (hunk 1 above)
Comment (yujinyuz) [reply src/courses/models.py]: done

**Comment by tylerecouture** [comment]
@coderabbitai review

**Pull request merged**